core/perf_test/ViewAllocationPerfTest.hpp

```diff
@@ -0,0 +1,34 @@
+#include <Kokkos_Core.hpp>
```

Comment:
Can you please elaborate on what `core/perf_test/ViewAllocationPerfTest.hpp ` tests and how?